build/lib/tests/backend_http/test_do_request.py

```diff
@@ -0,0 +1,77 @@
+import pytest
```

Comment:
Los tests deberían ir en la raíz. Ya hay una carpeta que se llama `tests/`, habría que meterlos ahí adentro.

Reply:
Yes, actually it looks I forgot to add this `build` dir in the `.gitignore` file.

Reply:
In this case this is done, since no tests outside the `tests/` dir in the root dir.